modules/serverless-rn-1-4-0.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-4-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.4.0
+
+[id="new-features-1-4-0_{context}"]
+== New features
+* {ServerlessProductName} 1.4.0 is available on {product-title} 4.2 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.11.0.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.11.0.
+* Knative Client is now available for download through the *Command Line Tools* page in the {product-title} web console.
+
```

Comment:
Let's please add a section that states that we've migrated our Knative Serving CR's APIgroup from `serving.knative.dev` to `operator.knative.dev`.

We can write it like this:

```
Note:
The KnativeServing object's api group has changed in this release from "serving.knative.dev" to 
"operator.knative.dev". This migration is done automatically when installing the operator. Please 
adjust all your scripts to reflect the new api group as shown in the "Installing Knative Serving" section.
```

I'll do a review of the existing docs and send a PR that moves existing occurrences over.